CO2_Gadget_EINK.h

```diff
@@ -17,8 +17,10 @@
 // clang-format on
 #include <GxEPD2_BW.h>
 
-uint16_t redrawDisplayEveryCycles = 10;  // Redraw display every X partial updates
-uint16_t cyclesLeftToRedrawDisplay = 0;  // Cycles left to redraw display
+// Variables for e-ink functionality -> move to platformio.ini or CO2_Gadget.ino in a future for menu config
```

Comment:
Hysteresis should go out of the e-Ink functionality as it will be globally used (other displays and also for, al least, WiFi, MQTT and ESPNow)